soknad/src/main/java/no/nav/k9/søknad/ytelse/psb/v1/PleiepengerSyktBarn.java

```diff
@@ -134,12 +126,12 @@ public Person getAnnenPart() {
         final List<Periode> perioder = new ArrayList<>(søknadsperiode);
         perioder.addAll(endringsperiode);
 
-        final var fom = søknadsperiode
+        final var fom = perioder
                 .stream()
                 .map(Periode::getFraOgMed)
                 .min(LocalDate::compareTo)
                 .orElseThrow();
-        final var tom = søknadsperiode
+        final var tom = perioder
```

Comment:
Om det finnes èn periode som har `tom=null` så skal vel tom bli null? Nå finner man den siste datoen også om det finnes en null?

Reply:
hmm, dette vil vel feile med "åpne" perioder ja

Reply:
Fom eller tom for psb skal ikke få lov til å være null